public/javascripts/routes-script.js

```diff
@@ -35,4 +35,18 @@ $(document).ready(function(){
         thisForm.closest('.media').remove();
       })
     });
+
+    $('.itinerary-delete').submit(function(e){
```

Comment:
This AJAX call appears not to work when the delete button is clicked.

Reply:
issue was resolved!